extra/man/alacritty.5.scd

```diff
@@ -61,6 +61,7 @@ override variables set by alacritty itself.
 ```
 [env]
 TERM = "alacritty"
+EGL_OVER_GLX = "true"
```

Comment:
Just use `PREFER_EGL=1` it's more common for `env` to use numbers.

Reply:
Should probably just add it to our debug section?

Reply:
Yeah, that as well.

Reply:
Numbers aren't currently supported it looks like. Is the string "1" ok?

Reply:
Do you mean that it should be added as a Debug option or just added to the section?